mentat/conversation.py

```diff
@@ -52,7 +55,25 @@ def get_model_response(self, config: ConfigManager) -> (str, list[CodeChange]):
         messages = self.messages.copy()
 
         code_message = self.code_file_manager.get_code_message()
-        messages.append({"role": "system", "content": code_message})
+        system_message = code_message
+
+        if self.code_map:
```

Comment:
This looks like an ideal situation for a walrus (:=) operator (instead of 2 nested if statements)